src/Operation/ListCollectionNames.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/*
+ * Copyright 2015-2017 MongoDB, Inc.
```

Comment:
I've updated new files with "2020-present". We may want to look into either changing the year for all files or omitting the second portion, as the copyright period extends from when the code was created to a certain time after the author's death (which may be more complicated since the Copyright is owned by MongoDB).

Reply:
See https://jira.mongodb.org/browse/DRIVERS-409 for context. I think that instructs using "-present" so we never have to update these again, and it also permits using the project inception date as the original date, although we can certainly use the current year as files are introduced.